src/test/java/com/teragrep/pth10/steps/teragrep/bloomfilter/TableSQLTest.java

```diff
@@ -66,6 +66,31 @@ public void testCreateTableSQLJournalDBOption() {
         Assertions.assertEquals(e, table.createTableSQL());
     }
 
+    @Test
+    public void testCreateTableSQLInvalidJournalDBOption() {
+        String name = "test_table";
+        TableSQL table = new TableSQL(name, "test;%00SELECT%00CONCAT('DROP%00TABLE%00IF%00EXISTS`',table_name,'`;')");
+        RuntimeException e = Assertions.assertThrows(RuntimeException.class, table::createTableSQL);
+        Assertions
+                .assertEquals(
+                        "malformed SQL input <[test;%00SELECT%00CONCAT('DROP%00TABLE%00IF%00EXISTS`',table_name,'`;')]>, only use alphabets, numbers and _",
+                        e.getMessage()
+                );
+    }
+
+    @Test
+    public void testCreateTableSQLInvalidJournalDBOptionIgnoreConstraints() {
+        String name = "test_table";
+        TableSQL table = new TableSQL(
+                name,
+                "String e = \"CREATE TABLE IF NOT EXISTS `test_table`(`id` BIGINT UNSIGNED NOT NULL auto_increment PRIMARY KEY,`partition_id` BIGINT UNSIGNED NOT NULL UNIQUE,`filter_type_id` BIGINT UNSIGNED NOT NULL,`filter` LONGBLOB NOT NULL,CONSTRAINT `test_table_ibfk_1` FOREIGN KEY (filter_type_id) REFERENCES filtertype (id)ON DELETE CASCADE,CONSTRAINT `test_table_ibfk_2` FOREIGN KEY (partition_id) REFERENCES journaldb.logfile (id)ON DELETE CASCADE);\";\n"
```

Comment:
This looks a bit weird here, is it intentional? With the assert equals in quotes

Reply:
cleared up